{"name": "test_mult[2-8-14]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 16 == 14\n  +16\n  -14", "trace": "self = <testcases.test_func.Test_Pytest object at 0x106657c50>, fixture_func = <pyCode.calc.Calculator object at 0x106657dd8>, a = 2, b = 8, result = 14\n\n    @pytest.mark.parametrize('a,b,result', test_data[2])\n    @allure.feature('乘法测试')\n    def test_mult(self, fixture_func, a, b, result):\n>       assert fixture_func.mult(a, b) == result\nE       assert 16 == 14\nE         +16\nE         -14\n\ntestcases/test_func.py:24: AssertionError"}, "attachments": [{"name": "stdout", "source": "a50cf835-286a-4d8f-8854-97cdd431ce42-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "a", "value": "2"}, {"name": "b", "value": "8"}, {"name": "result", "value": "14"}], "start": 1594373023546, "stop": 1594373023546, "uuid": "03476d5d-e37a-492f-ac42-4f8db8d82cb7", "historyId": "ab20fb020a30ec47adfec0c5a0e9d658", "testCaseId": "f23b8f69bb19ec69f89b4ceb1ecccfec", "fullName": "testcases.test_func.Test_Pytest#test_mult", "labels": [{"name": "feature", "value": "乘法测试"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_func"}, {"name": "subSuite", "value": "Test_Pytest"}, {"name": "host", "value": "localhost"}, {"name": "thread", "value": "51782-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_func"}]}